packages/markdown/src/marked.ts

```diff
@@ -150,9 +162,11 @@ export const apifyMarked = (markdown: string): MarkedResponse => {
     tokens.forEach((token) => {
         if (token.type === 'code' && token.lang) {
             if (token.lang === 'marked-tabs') {
-                codeTabsObjectPerIndex[markedTabTokenIndex] = codeTabObjectFromCodeTabMarkdown(token.text);
+                codeTabsObjectPerIndex[markedTabTokenIndex] =
```

Comment:
like this line, why are you breaking it? my guess is prettier and their dumb 80 chars max line length default? :]

Reply:
@B4nan yeah sorry that's prettier. I have auto-save when a window is unfocused

Reply:
But I have the line length limit to 140

Reply:
Using any auto-formatting on projects that does not provide configuration for the tool is... Let's say not very polite :] Even if we want to change code style, it can't be part of such a PR (that should have pretty much one line diff), it needs to be a separate one. 

> But I have the line length limit to 140

That line had 107 characters 🤷